### PR TITLE
added navigation menu and mobile responsiveness

### PR DESCRIPTION
**Pull request by SMakaiTakori**
added navigation menu and mobile responsiveness

### CODE REVIEW

**Comment by SMakaiTakori** [comment]
[Define/Build Site Navigation](https://trello.com/c/zvR3HW36/2-define-build-site-navigation)

File: src/components/navbar/navbar.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      <section className={styles.mobileTitle}>
+        <h1>Menu</h1>
+        <Navbar.Toggle
+          aria-controls="basic-navbar-nav"
+          className={styles.navToggle}
+        />
+      </section>
+      <Navbar.Collapse id="basic-navbar-nav" className={styles.navCollapse}>
+        <Nav>
+          <Nav.Link href="#" className={styles.navLink}>
```
Comment (SammySteiner) [inline src/components/navbar/navbar.js:34]: Do we have links for these yet?

**Review by SammySteiner — CHANGES_REQUESTED** [review]
Hey, great work so far. I know you're working on the icon switching, but here are a few things you should look into:

1. Width of 992px - 999px shows black background on desktop view links 
2. Width between 990px - 800px hamburger/menu should be right aligned
3. Width between 700px - 350px hamburger/menu should be right aligned
4. Should menu background be black and not grey?
5. Width of 1100px - 1000px shrinks the logo, can it disappear at 1100px instead?

File: src/components/navbar/navbar.js (hunk 1 above)
Comment (SMakaiTakori) [reply src/components/navbar/navbar.js:70]: We don't have links for these yet, last time I checked. I will ask Adam though.

Comment (SMakaiTakori) [reply src/components/navbar/navbar.js:70]: Also, the background on the figma designs opens up black below the gray.

**Review by jlhogan — APPROVED** [review]
LGTM, pending icon and link updates!
there's a conflict with the new v2 in `useDataMaps` so be sure to rebase or merge or however you like to resolve

**Pull request merged**